sale_stock_order_line_quantity_readonly/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+This module extends the functionality of the Sales module to restrict users from updating the quantity of products in sale order lines:
+
+- When the sale order is locked/confirmed, users are still able to adjust the quantities.
+- This becomes a problem if the related pickings are already (or partially) released, as it generates a return operation.
```

Comment:
Are you sure ? I've had the same problem but after install custom modules. That depends on the stock move merge process that uses some fields like 'date_deadline'

Reply:
@trisdoan he's right. You should mention that this is only a way to have more fine grained control and that it offers an hook to customize the behavior for custom modules.

Reply:
Hello @rousseldenis @simahawk, you're right. After checking with BA, Locked state is enough for them.